eth/tracers/testdata/state_diff_tracer_big_slow.json

```diff
@@ -0,0 +1,100 @@
+{
+  "genesis": {
+    "number": "553882",
+    "hash": "0xd1451849f99a9ce39b482aaca009bcba2e99be3b5e436f55ffb7869bb4d8a533",
+    "nonce": "0x47f15e3aa9bad4ec",
+    "mixHash": "0xcdd597f53eb00bb5c6ebf28a659d3608d9c08517857061ad166e55e2f40a35b1",
+    "stateRoot": "0x38d62b24b220d84ddd3a8e68aefead229e73a1a5d715ad33bf8f62bb8d7da0e3",
+    "miner": "0x877bd459c9b7d8576b44e59e09d076c25946f443",
+    "difficulty": "1971669",
+    "totalDifficulty": "459272439685",
+    "extraData": "0xd883010906846765746888676f312e31332e35856c696e7578",
+    "gasLimit": "6878269",
+    "timestamp": "1577401639",
+    "alloc": {
+      "0x877bd459c9b7d8576b44e59e09d076c25946f443": {
+        "balance": "0x17b561689d13557dfc3d",
+        "nonce": "7653",
+        "code": "0x",
+        "storage": {}
+      }
+    },
+    "config": {
```

Comment:
A note that using the Mordor config (both configuration format and configuration itself) for test cases limits their applicability to just core-geth; ethereumgo-ethereum won't be able to run these. 

Is it possible to use `&goethereum.ChainConfig` schemas here so they could be more easily shared upstream?

Reply:
Great point. Will have a look at this together with the rest stuff discussed yesterday. 
I believe this fits nicely on a new PR and doesn't worth blocking this PR at the moment. 
I agree, that it has to be done, before making an upstream PR.

Reply:
SGTM!

Reply:
Resolved by https://github.com/etclabscore/core-geth/pull/355/commits/208da5589f626f9200ec11f557b373dfc5645e40